src/block_misc.rs

```diff
@@ -1007,7 +1007,7 @@ mod tests {
             ]
         ); // backward part  (6+11)
```

Comment:
Also this comment (or expand into something that reflect 6+11's meaning exactly  maybe?)

Reply:
Removed.